src/test/java/uk/gov/hmcts/reform/draftstore/service/crypto/CryptoServiceTest.java

```diff
@@ -0,0 +1,45 @@
+package uk.gov.hmcts.reform.draftstore.service.crypto;
+
+import org.junit.Test;
+
+import java.util.stream.Stream;
+import javax.crypto.AEADBadTagException;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+
+public class CryptoServiceTest {
```

Comment:
Missing assertions for:
- plain == decrypted
- plain != encrypted

Reply:
added

Reply:
We could also add a test for checking if an exception would be thrown if we used an empty password or an empty input (although maybe we should allow for input to be empty) and a test checking if encrypted input of a very short length is long enough in the encrypted form.